CHANGES.rst

```diff
@@ -1121,6 +1121,7 @@ astropy.samp
 ^^^^^^^^^^^^
 
 - SAMP web profile CORS HTTP server implements `Private Network Access proposal <https://wicg.github.io/private-network-access>`_. [#16193]
+- samp_hub --log-level choices inferred from logging [#17670]
```

Comment:
Sorry if I wasn't clear. This file is not meant to be edited on regular PRs. Instead, you should create a new file under `docs/changes/samp/17673.bugfix.rst`

Reply:
Your CI is giving me a fight, thank you for the help! but I'm not sure what to do next.

Reply:
oops, resolved.